sentence_transformers/SentenceTransformer.py

```diff
@@ -10,6 +10,7 @@
 from numpy import ndarray
 import transformers
 from huggingface_hub import HfApi, HfFolder, Repository, hf_hub_url, cached_download
+from huggingface_hub.constants import HUGGINGFACE_HUB_CACHE
```

Comment:
I think this line is not needed. Could you remove it from the PR so that it can be merged?